lib/view.js

```diff
@@ -408,7 +408,8 @@ async function printData (data, name, opts) {
   log.disableProgress()
 
   // print directly to stdout to not unnecessarily add blank lines
-  console.log(msg.trim())
+  if (msg === '') process.stdout.write(msg.trim())
+  else console.log(msg.trim())
```

Comment:
Not gonna lie, I don't love this... but it does keep the test suite working. I'm not 100% if we actually want that newline for all other test cases. I can revert 37fb60b26 and update the snapshots if we want to "no newline" behavior for all cases

Reply:
I just double checked output locally and it would appear that it is only the error case that has the additional newline right now

Reply:
Writing `''` to process.stdout is a no-op.  This could be just:

```js
if (msg !== '') {
  console.log(msg)
}
```